doc/releases/release-notes-3.5.rst

```diff
@@ -148,6 +148,10 @@ Boards & SoC Support
 Build system and infrastructure
 *******************************
 
+* Twister now supports ``required_snippets`` in testsuite .yml files, this can
+  be used to include a snippet when a test is ran (and exclude any boards from
+  running that the snippet cannot be applied to).
```

Comment:
pleaes update the twister document and provide some examples on usage

Reply:
done